<!doctype html>
<html lang="en">
<head>
  <title>Gene Info eXtension (GIX)</title>
  <meta name="Description" content="Gene Information eXtension (GIX) is a browser extension for retrieving gene information on any webpage.">
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link rel="apple-touch-icon" sizes="180x180" href="/favicon/apple-touch-icon.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/favicon/favicon-32x32.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/favicon/favicon-16x16.png">
  <link rel="manifest" href="/site.webmanifest">
  <link rel="mask-icon" href="/favicon/safari-pinned-tab.svg" color="#658170">
  <meta name="msapplication-TileColor" content="#fdfdfd">
  <meta name="theme-color" content="#658170">
  <!--analytics-->

  <link rel="stylesheet" href="/assets/style.css">
</head>
<body>
  <nav class="nav no-print">
    <div class="nav-left">
      <img alt="GIX logo" src="/assets/gix.svg" height="35px">
      <span>GIX</span>
    </div>
    <div class="nav-right">
      <a class="nav-right__workspace" href="/workspace">
        Workspace
      </a>
      <a
        href="https://github.com/knightjdr/gene-info"
        tooltip="GitHub"
        tooltip-position="bottom"
      >
        <svg
          xmlns="http://www.w3.org/2000/svg"
          aria-label="GitHub code repository"
          role="img"
          height="35"
          viewBox="0 0 645.188 629.067"
        >
          <title>GitHub</title>
          <desc>code repository</desc>
          <path d="M215.8 506.524c0 2.602-2.992 4.683-6.764 4.683-4.292.391-7.285-1.691-7.285-4.683 0-2.602 2.992-4.683 6.764-4.683 3.902-.39 7.285 1.691 7.285 4.683zm-40.455-5.853c-.911 2.602 1.691 5.593 5.593 6.374 3.382 1.301 7.285 0 8.065-2.602.781-2.602-1.691-5.594-5.593-6.765-3.382-.909-7.154.391-8.065 2.993zm57.495-2.212c-3.772.911-6.374 3.383-5.983 6.374.39 2.602 3.772 4.293 7.674 3.382 3.772-.91 6.374-3.382 5.984-5.983-.391-2.472-3.903-4.163-7.675-3.773zM318.432 0C138.013 0 0 136.972 0 317.391c0 144.257 90.794 267.701 220.482 311.146 16.65 2.992 22.503-7.284 22.503-15.739 0-8.064-.39-52.552-.39-79.868 0 0-91.055 19.512-110.176-38.763 0 0-14.829-37.853-36.162-47.608 0 0-29.788-20.423 2.081-20.032 0 0 32.39 2.602 50.21 33.56 28.487 50.211 76.226 35.771 94.827 27.187 2.992-20.812 11.447-35.251 20.812-43.836-72.713-8.065-146.078-18.602-146.078-143.736 0-35.771 9.886-53.723 30.699-76.616-3.382-8.455-14.439-43.316 3.382-88.323 27.187-8.455 89.754 35.121 89.754 35.121 26.016-7.284 53.982-11.057 81.689-11.057 27.706 0 55.673 3.772 81.688 11.057 0 0 62.567-43.706 89.754-35.121 17.821 45.137 6.765 79.868 3.382 88.323 20.812 23.024 33.561 40.975 33.561 76.616 0 125.525-76.616 135.541-149.33 143.736 11.968 10.276 22.113 29.788 22.113 60.356 0 43.836-.39 98.078-.39 108.745 0 8.455 5.983 18.731 22.503 15.739 130.078-43.186 218.271-166.63 218.271-310.887C645.188 136.972 498.85 0 318.432 0zM126.436 448.64c-1.691 1.301-1.301 4.292.911 6.764 2.081 2.081 5.073 2.992 6.764 1.301 1.691-1.301 1.301-4.292-.91-6.764-2.082-2.082-5.074-2.993-6.765-1.301zm-14.048-10.536c-.911 1.69.39 3.771 2.992 5.072 2.081 1.301 4.683.911 5.593-.91.911-1.691-.39-3.772-2.992-5.073-2.602-.781-4.683-.39-5.593.911zm42.145 46.307c-2.081 1.69-1.301 5.593 1.691 8.064 2.992 2.992 6.764 3.382 8.455 1.301 1.691-1.69.911-5.593-1.691-8.064-2.862-2.992-6.764-3.382-8.455-1.301zm-14.829-19.122c-2.082 1.301-2.082 4.684 0 7.675 2.081 2.992 5.593 4.293 7.284 2.992 2.081-1.691 2.081-5.073 0-8.065-1.821-2.992-5.203-4.292-7.284-2.602z"/>
        </svg>
      </a>
    </div>
  </nav>
  <nav class="side-nav">
    <div class="side-nav__inner">
      <ul class="side-nav__links">
        <li><a href="#video">Video tutorial</a></li>
        <li><a href="#settings">Settings</a></li>
        <li><a href="#options">Information options</a></li>
        <li><a href="#panel">Information panel</a></li>
        <li><a href="#database">Database</a></li>
        <li><a href="#features">Feature requests</a></li>
        <li><a href="#unsupported">Unsupported websites</a></li>
        <li><a href="#permissions">Permissions</a></li>
        <li><a href="#logging_privacy">Logging & Privacy</a></li>
        <li><a href="#source">Source code</a></li>
        <li><a href="#issues">Issues</a></li>
        <li><a href="#browser">Browser support</a></li>
        <li><a href="#manuscript">Manuscript & citation</a></li>
        <li><a href="#funding">Funding</a></li>
        <li><a href="/workspace">Workspace</a></li>
      </ul>
      <hr>
      <small>
        version 1.8
      </small>
    </div>
  </nav>
  <div class="documentation">
    <header>
      <h1>Gene Information eXtension (GIX)</h1>
      <!-- <p class="outage-notice">
        OUTAGE: GIX will be unavailable for use from 9am - 5pm EST Monday Feb 14 due to maintenance
        at our cloud provider.
      </p> -->
      <p>
        GIX is a browser extension that allows you to retrieve information about
        a gene product directly on any webpage simply by double clicking an official
        gene name, synonym or supported accession.
      </p>
      <p>
        This extension is developed by the laboratory of
        <a href="https://gingraslab.org">Dr. Anne-Claude Gingras</a>
        at the Lunenfeld-Tanenbaum Research Institute in Toronto, Canada.
      </p>
      <p class="no-print">
        <h2>Download</h2>
        <a class="link-button" href="https://chrome.google.com/webstore/detail/gene-info/jggendahejbhkghnachhlomkkheomchp">
          <svg xmlns="http://www.w3.org/2000/svg" height="20" viewBox="10.406 1 645.179 645.152">
            <path d="M181.459 273.521L82.079 120.81C143.997 43.804 236.872 1.398 331.829 1.008c55.023-.39 111.217 13.658 162.338 43.186 56.454 32.78 99.38 79.868 126.696 133.98l-267.051-14.048c-75.575-4.423-147.508 38.113-172.353 109.395zm42.796 50.081c0 60.096 48.649 108.745 108.745 108.745 60.096 0 108.745-48.649 108.745-108.745 0-60.096-48.649-108.745-108.745-108.745-60.096-.001-108.745 48.518-108.745 108.745zm409.616-116.03l-181.72 9.366c49.3 57.625 50.08 140.744 8.585 204.482L314.919 645.675c60.486 3.252 122.794-10.016 179.248-42.796 139.704-80.649 196.288-249.75 139.704-395.307zm-449.55 177.947L62.958 146.825c-33.17 50.73-52.552 111.607-52.552 176.776 0 161.297 118.111 294.887 272.514 318.562l82.859-162.338c-74.925 14.049-147.248-27.055-181.458-94.306z"/>
          </svg>
          Chrome
        </a>
        <a class="link-button" href="https://addons.mozilla.org/addon/gene-info/">
          <svg xmlns="http://www.w3.org/2000/svg" height="20" viewBox="0 0 624.375 643.887">
            <path d="M621.903 294.237c-.91-5.854-1.821-9.235-1.821-9.235s-2.341 2.602-6.113 7.674c-1.171-13.918-3.643-27.576-7.545-41.104-4.812-16.78-11.057-33.04-18.861-48.649-4.942-10.406-10.666-20.292-17.3-29.658-2.342-3.512-4.813-7.024-7.284-10.276-11.447-18.731-24.715-30.309-39.935-52.031-9.886-16.65-16.78-34.991-20.032-54.113-4.162 11.577-7.414 23.414-9.625 35.511-15.74-15.869-29.268-27.056-37.593-34.73C415.47 19.642 420.152 0 420.152 0s-75.836 84.681-43.056 172.874c11.316 29.918 30.958 56.063 56.454 75.315 31.738 26.275 66.079 46.828 84.16 99.64-14.568-27.707-36.552-50.991-63.479-66.99 8.065 19.122 12.228 39.804 12.098 60.486 0 79.348-64.519 143.736-143.866 143.606-10.797 0-21.463-1.171-31.869-3.643a134.18 134.18 0 0 1-35.642-12.097c-16.78-10.146-31.219-23.545-42.666-39.414l-.26-.391 2.602.911c5.983 2.081 11.967 3.642 18.211 4.812 24.325 5.203 49.82 2.212 72.323-8.585 22.764-12.617 36.422-21.983 47.608-18.211h.261c10.927 3.512 19.512-7.154 11.707-18.211-13.528-17.43-35.642-26.016-57.495-22.113-22.763 3.252-43.576 19.512-73.364 3.772-1.951-1.04-3.772-2.081-5.593-3.252-2.081-1.171 6.374 1.69 4.423.39-6.504-3.252-12.748-7.023-18.731-11.187-.39-.39 4.553 1.431 4.032 1.041-7.674-5.203-14.309-11.968-19.512-19.772-5.333-9.625-5.854-21.333-1.301-31.349 2.732-4.943 7.024-8.975 12.098-11.316 3.902 1.951 6.244 3.382 6.244 3.382s-1.691-3.252-2.732-4.943c.391-.13.65 0 1.041-.26 3.382 1.431 10.796 5.203 14.829 7.544 2.732 1.431 4.943 3.512 6.764 6.114 0 0 1.301-.65.39-3.512-1.431-3.512-3.772-6.504-7.024-8.585h.26c2.992 1.561 5.854 3.382 8.585 5.333a41.655 41.655 0 0 0 3.382-18.211c.26-3.382-.26-6.894-1.431-10.146-1.041-2.081.65-2.862 2.471-.65-.26-1.691-.91-3.252-1.561-4.813v-.13s1.041-1.431 1.561-1.951a23.866 23.866 0 0 1 4.423-3.512 191.138 191.138 0 0 1 29.528-15.089c8.325-3.643 15.219-6.374 16.65-7.285 2.081-1.301 4.032-2.862 5.854-4.553 6.895-5.854 11.707-14.048 13.268-23.024.131-1.17.261-2.341.391-3.642v-1.951c-1.171-4.553-8.976-7.935-49.95-11.837-14.438-2.341-26.016-13.138-29.268-27.447v-.13a104.132 104.132 0 0 1 40.064-49.82c1.041-.911-4.163.26-3.122-.65 3.512-1.691 7.024-3.252 10.667-4.553 1.821-.781-7.805-4.423-16.39-3.512-5.203.26-10.406 1.561-15.219 3.642 2.081-1.691 8.064-4.032 6.634-4.032-10.927 2.081-21.463 6.114-31.089 11.707 0-1.041.13-1.951.65-2.862-7.674 3.252-14.309 8.455-19.512 14.959.13-1.171.26-2.341.26-3.512-3.512 2.602-6.764 5.593-9.496 8.976l-.13.13c-22.634-8.715-47.219-10.796-71.023-6.114l-.26-.13h.26c-4.943-4.032-9.235-8.715-12.618-14.179l-.26.13-.521-.26c-1.561-2.341-3.122-4.943-4.813-7.805-1.17-2.081-2.341-4.423-3.512-6.764a.28.28 0 0 0-.26-.26c-.521 0-.781 2.211-1.171 1.691v-.13c-4.163-10.796-6.114-22.374-5.724-34.081l-.26.13c-6.634 4.553-11.707 11.187-14.439 18.861-1.17 2.731-2.081 4.292-2.862 5.854v-.65c.13-1.431.781-4.292.65-4.033-.13.26-.26.391-.39.521-1.951 2.211-3.772 4.813-5.073 7.544-1.171 2.472-2.211 5.073-2.992 7.675-.13.39 0-.39 0-1.301s.13-2.602 0-2.211l-.39.911A192.31 192.31 0 0 0 54.5 164.157c-.521 3.642-.781 7.285-.65 10.796v.26c-6.244 6.764-11.707 14.309-16.52 22.244-15.74 26.536-27.447 55.283-34.861 85.331a207.274 207.274 0 0 1 18.601-32.649A301.81 301.81 0 0 0 0 360.967c2.341-11.187 5.463-22.113 9.105-32.909-2.211 44.877 6.374 89.623 25.235 130.468 25.235 56.584 67.12 104.062 120.062 136.192 21.593 14.568 45.137 25.885 69.982 33.56a872.753 872.753 0 0 0 10.016 3.513c-1.041-.391-2.082-.911-3.122-1.301 29.398 8.845 60.096 13.397 90.794 13.397 108.876 0 144.777-41.495 148.029-45.527 5.333-4.812 9.756-10.666 12.878-17.3a312.844 312.844 0 0 1 6.373-2.732l1.301-.65 2.472-1.17c16.39-7.675 31.869-17.431 45.918-28.748 21.202-15.219 36.292-37.332 42.796-62.567 3.902-9.235 4.032-19.512.52-28.877 1.171-1.821 2.212-3.643 3.513-5.594 23.414-37.593 36.682-80.519 38.503-124.745v-3.642c0-9.497-.78-18.862-2.472-28.098zM232.97 167.281c-.52 1.431-1.17 2.992-1.691 4.553a63.869 63.869 0 0 1 1.691-4.553z"/>
          </svg>
          Firefox
        </a>
        <a class="link-button" href="https://microsoftedge.microsoft.com/addons/detail/gene-info/hegihjieeonojcnhfmhmedmoobggamka">
          <svg xmlns="http://www.w3.org/2000/svg" height="20" viewBox="0 0 512 512">
          <path d="M481.92,134.48C440.87,54.18,352.26,8,255.91,8,137.05,8,37.51,91.68,13.47,203.66c26-46.49,86.22-79.14,149.46-79.14,79.27,0,121.09,48.93,122.25,50.18,22,23.8,33,50.39,33,83.1,0,10.4-5.31,25.82-15.11,38.57-1.57,2-6.39,4.84-6.39,11,0,5.06,3.29,9.92,9.14,14,27.86,19.37,80.37,16.81,80.51,16.81A115.39,115.39,0,0,0,444.94,322a118.92,118.92,0,0,0,58.95-102.44C504.39,176.13,488.39,147.26,481.92,134.48ZM212.77,475.67a154.88,154.88,0,0,1-46.64-45c-32.94-47.42-34.24-95.6-20.1-136A155.5,155.5,0,0,1,203,215.75c59-45.2,94.84-5.65,99.06-1a80,80,0,0,0-4.89-10.14c-9.24-15.93-24-36.41-56.56-53.51-33.72-17.69-70.59-18.59-77.64-18.59-38.71,0-77.9,13-107.53,35.69C35.68,183.3,12.77,208.72,8.6,243c-1.08,12.31-2.75,62.8,23,118.27a248,248,0,0,0,248.3,141.61C241.78,496.26,214.05,476.24,212.77,475.67Zm250.72-98.33a7.76,7.76,0,0,0-7.92-.23,181.66,181.66,0,0,1-20.41,9.12,197.54,197.54,0,0,1-69.55,12.52c-91.67,0-171.52-63.06-171.52-144A61.12,61.12,0,0,1,200.61,228,168.72,168.72,0,0,0,161.85,278c-14.92,29.37-33,88.13,13.33,151.66,6.51,8.91,23,30,56,47.67,23.57,12.65,49,19.61,71.7,19.61,35.14,0,115.43-33.44,163-108.87A7.75,7.75,0,0,0,463.49,377.34Z"/>
          </svg>
          Edge
        </a>
        <a class="link-button" href="https://apps.apple.com/us/app/gix-for-safari/id1543199063">
          <svg xmlns="http://www.w3.org/2000/svg" height="20" viewBox="0 0 512 512">
            <path d="M274.69,274.69l-37.38-37.38L166,346ZM256,8C119,8,8,119,8,256S119,504,256,504,504,393,504,256,393,8,256,8ZM411.85,182.79l14.78-6.13A8,8,0,0,1,437.08,181h0a8,8,0,0,1-4.33,10.46L418,197.57a8,8,0,0,1-10.45-4.33h0A8,8,0,0,1,411.85,182.79ZM314.43,94l6.12-14.78A8,8,0,0,1,331,74.92h0a8,8,0,0,1,4.33,10.45l-6.13,14.78a8,8,0,0,1-10.45,4.33h0A8,8,0,0,1,314.43,94ZM256,60h0a8,8,0,0,1,8,8V84a8,8,0,0,1-8,8h0a8,8,0,0,1-8-8V68A8,8,0,0,1,256,60ZM181,74.92a8,8,0,0,1,10.46,4.33L197.57,94a8,8,0,1,1-14.78,6.12l-6.13-14.78A8,8,0,0,1,181,74.92Zm-63.58,42.49h0a8,8,0,0,1,11.31,0L140,128.72A8,8,0,0,1,140,140h0a8,8,0,0,1-11.31,0l-11.31-11.31A8,8,0,0,1,117.41,117.41ZM60,256h0a8,8,0,0,1,8-8H84a8,8,0,0,1,8,8h0a8,8,0,0,1-8,8H68A8,8,0,0,1,60,256Zm40.15,73.21-14.78,6.13A8,8,0,0,1,74.92,331h0a8,8,0,0,1,4.33-10.46L94,314.43a8,8,0,0,1,10.45,4.33h0A8,8,0,0,1,100.15,329.21Zm4.33-136h0A8,8,0,0,1,94,197.57l-14.78-6.12A8,8,0,0,1,74.92,181h0a8,8,0,0,1,10.45-4.33l14.78,6.13A8,8,0,0,1,104.48,193.24ZM197.57,418l-6.12,14.78a8,8,0,0,1-14.79-6.12l6.13-14.78A8,8,0,1,1,197.57,418ZM264,444a8,8,0,0,1-8,8h0a8,8,0,0,1-8-8V428a8,8,0,0,1,8-8h0a8,8,0,0,1,8,8Zm67-6.92h0a8,8,0,0,1-10.46-4.33L314.43,418a8,8,0,0,1,4.33-10.45h0a8,8,0,0,1,10.45,4.33l6.13,14.78A8,8,0,0,1,331,437.08Zm63.58-42.49h0a8,8,0,0,1-11.31,0L372,383.28A8,8,0,0,1,372,372h0a8,8,0,0,1,11.31,0l11.31,11.31A8,8,0,0,1,394.59,394.59ZM286.25,286.25,110.34,401.66,225.75,225.75,401.66,110.34ZM437.08,331h0a8,8,0,0,1-10.45,4.33l-14.78-6.13a8,8,0,0,1-4.33-10.45h0A8,8,0,0,1,418,314.43l14.78,6.12A8,8,0,0,1,437.08,331ZM444,264H428a8,8,0,0,1-8-8h0a8,8,0,0,1,8-8h16a8    ,8,0,0,1,8,8h0A8,8,0,0,1,444,264Z"/>
          </svg>
          Safari
        </a>
      </p>
    </header>
    <main>
      <section class="documentation__links no-print">
        <h2>Contents</h2>
        <ul>
          <li><a href="#video">Video tutorial</a></li>
          <li><a href="#settings">Settings</a></li>
          <li><a href="#options">Information options</a></li>
          <li><a href="#panel">Information panel</a></li>
          <li><a href="#database">Database</a></li>
          <li><a href="#features">Feature requests</a></li>
          <li><a href="#unsupported">Unsupported websites</a></li>
          <li><a href="#permissions">Permissions</a></li>
          <li><a href="#logging_privacy">Logging & privacy</a></li>
          <li><a href="#source">Source code</a></li>
          <li><a href="#issues">Issues</a></li>
          <li><a href="#browser">Browser support</a></li>
          <li><a href="#manuscript">Manuscript & citation</a></li>
          <li><a href="#funding">Funding</a></li>
          <li><a href="/workspace">Workspace</a></li>
        </ul>
      </section>
      <section class="new-features">
        <h2>New features (version 1.8.0)</h2>
        <ul>
          <li>
            Added an automated check for any outages at our cloud provider,
            with a notification displaying their impact on GIX
          </li>
        </ul>
        <a href="/changelog">full changelog</a>
      </section>
      <section class="no-print">
        <span class="anchor" id="video"></span>
        <h2>Video tutorial</h2>
        <video controls preload="metadata">
          <source src="/assets/tutorial.mp4" type="video/mp4">
          Sorry, your browser doesn't support embedded videos.
        </video>
      </section>
      <section>
        <span class="anchor" id="settings"></span>
        <h2>Settings</h2>
        <h3>Activation method</h3>
        <p>
          The default method for querying a gene is to double click on it. You can instead
          choose the drag method that requires you to mouse down on one side of the gene
          name, drag the selection box to the other side and release the mouse button. This
          can be useful on websites where double clicking selects more that one word (this
          is a rare occurrence and depends on the markup used by the website). The drag
          method is also necessary for gene names that have certain special characters in
          them. For example, double clicking a gene name with a dash '-' will only capture
          text before or after the dash but not including it.
        </p>
        <p>
          If you enable the "CTRL/⌘ required" option then double-clicking
          (or dragging) on its own will not initiate a query. On Linux and Windows systems
          you will have to hold the CTRL key while clicking to perform a query, or on Mac
          it is the Command/⌘ key. This restriction can ensure that the extension only activates
          when you want, rather than whenever you double-click.
        </p>
        <p>
          If you wish to completely disable the extension, simply click the disable button.
        </p>
        <h3>Display options</h3>
        <p>
          The extension can display information in two ways:
          <ol>
            <li>
              Detailed report: a panel spanning the full height of the browser window
              displaying links to external resources and the information from those resources.
            </li>
            <li>
              Tooltip report: a small box that opens at the point of the mouse click with
              only links to external resources.
            </li>
          </ol>
        </p>
        <h3>Species</h3>
        <p>
          In addition to human gene information our database contains information for
          the following model organisms:
        </p>
        <ul>
          <li><i>Arabidopsis thaliana</i></li>
          <li><i>Caenorhabditis elegans</i></li>
          <li><i>Danio rerio</i></li>
          <li><i>Dictyostelium discoideum</i></li>
          <li><i>Drosophila melanogaster</i></li>
          <li><i>Escherichia coli (strain K12)</i></li>
          <li><i>Gallus gallus</i></li>
          <li><i>Mus musculus</i></li>
          <li><i>Saccharomyces cerevisiae</i></li>
          <li><i>Salmonella Typhimurium (strain LT2)</i></li>
          <li><i>Schizosaccharomyces pombe</i></li>
          <li><i>Xenopus laevis</i></li>
        </ul>
        <p>
          Some information will only be available for certain species. For instance, Human
          Protein Atlas localization and expression information is only available for human
          gene entries. The Compartments subcellular localization database has information
          on six of the organisms we support:
          <i>Arabidopsis</i>,
          <i>C. elegans</i>,
          <i>Drosophila</i>,
          human,
          mouse
          and yeast.
        </p>
        <h3>Query type</h3>
        <p>
          The extension can retrieve information for any of the following query types:
        </p>
        <ul>
          <li>Ensembl gene (e.g. ENSG00000114209)</li>
          <li>Ensembl protein (e.g. ENSP00000376506)</li>
          <li>Gene name (e.g. PDCD10)</li>
          <li>NCBI Entrez gene ID (e.g. 11235)</li>
          <li>RefSeq (either nucleotide or protein e.g. NM_007217.3, NP_009148.2)</li>
          <li>UniProt (e.g. Q9BUL8)</li>
        </ul>
        <p>
          By default it will autodetect the query type but you can disable this option and
          force it to treat all queries as a specific type.
        </p>
        <p>
          Genes may also have a locus and ORF associated with them that can be used for querying.
          Loci were specifically included for yeast and ORFs for flies and worms to enable
          querying by commonly used nomenclature of the following formats:
        </p>
        <ul>
          <li>YGL190C (yeast locus for gene CDC55)</li>
          <li>CG12217 (fly ORF for gene PpV)</li>
          <li>C14A4.11 (worm ORF for gene ccm-3)</li>
        </ul>
        <p>
          You can query by locus or ORF for other species by selecting the appropriate option from
          the settings dropdown, however it will only autodetect these query types in yeast, flies
          and worms as these species adhere to specific naming formats that do not necessarily hold
          in other species.
        </p>
        <p>
          Some organism-specific selectors can also be automatically detected:
        </p>
        <ul>
          <li>neXtProt, <i>H. sapiens</i> (e.g. NX_Q9BUL8)</li>
          <li>Wormbase, <i>C. elegans</i> (e.g. WBGene00007561)</li>
        </ul>
        <h3>Search</h3>
        <p>
          The search box can be used for making queries in addition to using the mouse.
          Simply type in a query, hit enter and the information panel will appear on the
          current web page. Unlike mouse events, the search box will work on websites
          with embedded content like Google Docs and PDF viewers.
        </p>
      </section>
      <section>
        <span class="anchor" id="options"></span>
        <h2>Information options</h2>
        <p>
          Each pierce of information provided by GIX can be toggled off in the extension
          popup and will be hidden on subsequent queries.
        </p>
        <p>
          The GO term, Localization, Protein expression and RNA expression sections have advanced display
          options that can be viewed by clicking the settings/ellipsis button
          (<svg class="ellipsis" xmlns="http://www.w3.org/2000/svg" height="15" viewBox="0 0 512 512">
            <path fill="#658170" d="M328 256c0 39.8-32.2 72-72 72s-72-32.2-72-72 32.2-72 72-72 72 32.2 72 72zm104-72c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72zm-352 0c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72z" />
          </svg>)
          next to the relevant field in the extension popup. For GO terms, you can set the default
          namespace to display on the information panel: BP (biological process), CC
          (cellular compartment) or MF (molecular function). You can choose which
          resources to display for localization, including the Compartments subcellular
          localization database, the Human Protein Atlas and/or UniProt. Under Protein and RNA expression
          you can choose which cell lines and tissues to display by default in the information
          panel. There are 135 cell lines and 58 tissues for protein expression,
          and 64 cell lines and 37 tissues for RNA.
        </p>
        <p>
          You can control the order in which information appears on the results panel
          by dragging and dropping the settings in the extension popup.
        </p>
        <p>
          Certain data is only available for specific species as outlined above
          in the <strong>Species</strong> section.
        </p>
      </section>
      <section>
        <span class="anchor" id="panel"></span>
        <h2>Information panel</h2>
        <h3>Multiple results</h3>
        <p>
          If a query matches to multiple genes, as can happen since gene names are not always unique,
          the information panel will
          display one result and the others will be available from the dropdown menu
          that appears next to the <strong>Gene</strong> heading. A match to an
          official gene name will always be selected as the initial result to display
          in the case of multiple matches.
        </p>
        <h3>Buttons</h3>
        <ul class="buttons">
          <li>
            <img alt="Close button" src="/assets/close-button.svg">
            close the extension
          </li>
          <li>
            <img alt="Drag button" src="/assets/drag-button.svg">
            reposition the extension horizontally
          </li>
          <li>
            <img alt="Back button" src="/assets/back-button.svg">
            go back to the previous result
          </li>
          <li>
            <img alt="Resize button" src="/assets/resize-button.svg">
            resize the extension (bottom left corner)
          </li>
        </ul>
      </section>
      <section>
        <span class="anchor" id="database"></span>
        <h2>Database</h2>
        <p>
          The database will be updated on the 15th of each month or the next
          business day if the 15th falls on a weekend or holiday.
        </p>
        <h3>Integrated resources</h3>
        <p>
          We integrate several databases to generate the database used for GIX.
          The resources, urls and citations are listed below in alphabetical order:
        </p>
        <ul>
          <li>
            BioGRID,
            <a href="https://thebiogrid.org">thebiogrid.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/16381927">PMID:16381927</a>)
          </li>
          <li>
            Compartments,
            <a href="https://compartments.jensenlab.org">compartments.jensenlab.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/24573882">PMID:24573882</a>)
          </li>
          <li>
            DepMap,
            <a href="https://depmap.org">depmap.org</a>
            (<a href="https://pubmed.ncbi.nlm.nih.gov/29083409">PMID:29083409</a>)
          </li>
          <li>
            Gene Ontology,
            <a href="http://www.geneontology.org">www.geneontology.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/10802651">PMID:10802651</a>,
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/27899567">PMID:27899567</a>)
          </li>
          <li>
            HGNC,
            <a href="https://www.genenames.org">www.genenames.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/27799471">PMID:27799471</a>)
          </li>
          <li>
            Human Protein Atlas,
            <a href="https://www.proteinatlas.org">www.proteinatlas.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/25613900">PMID:25613900</a>,
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/28495876">PMID:28495876</a>)
          </li>
          <li>
            IntAct,
            <a href="https://www.ebi.ac.uk/intact">www.ebi.ac.uk/intact</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/24234451">PMID:24234451</a>)
          </li>
          <li>
            OMIM,
            <a href="https://www.omim.org">www.omim.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/30445645">PMID:30445645</a>)
          </li>
          <li>
            Pfam,
            <a href="https://pfam.xfam.org">pfam.xfam.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/30357350">PMID:30357350</a>)
          </li>
          <li>
            ProteomicsDB,
            <a href="https://www.proteomicsdb.org/">proteomicsdb.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/31665479">PMID:31665479</a>)
          </li>
          <li>
            Reactome,
            <a href="https://reactome.org">reactome.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/29145629">PMID:29145629</a>)
          </li>
          <li>
            UniProt,
            <a href=" https://www.uniprot.org">www.uniprot.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/29425356">PMID:29425356</a>)
          </li>
        </ul>
      </section>
      <section>
        <span class="anchor" id="features"></span>
        <h2>Feature requests</h2>
        <p>
          If you would like to request a new feature or the addition of a species,
          please use our <a href="https://github.com/knightjdr/gene-info/issues">issue tracker</a>.
          You can make requests via the Chrome Web Store's support page but unfortunately
          developers do not receive e-mail notifications when that happens and users do not
          receive notifications when we reply, so GitHub is the preferred place to make requests.
        </p>
        <p>
          Alternatively if you would like to add a feature yourself, please fork the code
          on <a href="https://github.com/knightjdr/gene-info">GitHub</a> and submit a
          pull request with your changes.
        </p>
      </section>
      <section>
        <span class="anchor" id="unsupported"></span>
        <h2>Unsupported websites</h2>
        <p>
          The extension is not able to capture mouse events (double clicks, etc.)
          on websites with embedded elements like Google Docs and PDF viewers and you
          can not therefore perform queries in the default way. The search box in the extension
          popup can however still be used for performing queries on these sites.
        </p>
        <p>
          Browsers block extensions from running on the tab homepage - the page you see when
          opening a new tab - as well as a few other privileged pages, and so neither double
          clicking nor the search bar will work in these contexts. The extension will alert
          you to this with a warning at the bottom of the popup.
        </p>
      </section>
      <section>
        <span class="anchor" id="permissions"></span>
        <h2>Permissions</h2>
        <p>
          When you install the extension it will alert you that it can "Read and change
          all your data on the websites you visit" on Chrome or "Access your data for
          all websites" in Firefox.
        </p>
        <p>
          To capture text from websites you visit this extension needs to be granted
          permission to read the sites. The only thing this extension reads is the
          text selected via double clicking or mouse dragging (if you enable that
          option).
        </p>
        <p>
          Adding an information panel or tooltip to a webpage with information about the gene you have
          selected requires permission to modify the page. The only thing we modify about the websites
          you visit is the addition of the panel or tooltip.
        </p>
        <p>
          If you would like to further restrict the access given to this extension in Chrome,
          you can right click on the icon in your browser toolbar and scroll to
          "This Can Read and Change Site Data" to view the options. Detailed
          instructions can be found at the
          <a href="https://support.google.com/chrome_webstore/answer/2664769?p=enable_extensions&visit_id=636834215946361040-2354360077&rd=1#extensionpermissions">
            Web Store's Help Page</a>.
        </p>
      </section>
      <section>
        <span class="anchor" id="logging_privacy"></span>
        <h2>Logging & privacy policy</h2>
        <p>
          We log three pieces of information with each query: 1) whether it matched to
          something in our database; 2) the species; and 3) the type of query, i.e. was
          it a gene name, UniProt ID, etc. We do not log the actual query term itself.
          Recording the total number of queries allows us to ensure the server has
          enough hardware resources to meet the demands of the user base. Recording the
          species allows us to prioritize development time and add resources for the
          most used species. For example, if a model organism is only searched by 0.01%
          of the user base, it will get less priority than one used by 95% of users. Finally,
          recording the type of query allows us to determine whether it is worth continuing to
          support a particular type of accession.
        </p>
        <p>
          We do not collect, store or share any personal information or IP addresses and the
          extension does not use analytics.
        </p>
      </section>
      <section>
        <span class="anchor" id="source"></span>
        <h2>Source code</h2>
        <p>
          Source code is available at
          <a href="https://github.com/knightjdr/gene-info/">GitHub</a>.
        </p>
        <p>
      </section>
      <section>
        <span class="anchor" id="issues"></span>
        <h2>Issues</h2>
        <p>
          If you find the extension does not work for you it may be that you are located
          behind a very strict firewall that blocks the method we are using to fetch data
          from our server (XMLHttpRequest) or that blocks the requests but not the method
          specifically. If this is the case, you'll have to discuss
          the matter with your IT department. If we can help to resolve the issue, please
          contact us.
        </p>
        <p>
          If you would like to report a bug, request a feature or have your resource
          added to GIX, please use our
          <a href="https://github.com/knightjdr/gene-info/issues">issue tracker</a>.
        </p>
      </section>
      <section>
        <span class="anchor" id="browser"></span>
        <h2>Browser support</h2>
        <div class="browser-support">
          <table>
            <thead>
              <tr>
                <th>Browser</th>
                <th>Supported</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>Brave</td>
                <td>&#x2714;</td>
              </tr>
              <tr>
                <td>Chrome</td>
                <td>&#x2714;</td>
              </tr>
              <tr>
                <td>Edge</td>
                <td>&#x2714;</td>
              </tr>
              <tr>
                <td>Firefox</td>
                <td>&#x2714;</td>
              </tr>
              <tr>
                <td>Opera</td>
                <td>&#x2714;</td>
              </tr>
              <tr>
                <td>Safari</td>
                <td>&#x2714;</td>
              </tr>
            </tbody>
          </table>
        </div>
        <h3>Firefox</h3>
        <p>
          GIX should work in Firefox as of version 57.0 (2017-11-14) but please update to the latest
          version to ensure full compatibility.
        </p>
        <h3>Microsoft Edge</h3>
        <p>
          To add the extension to Edge, first ensure you have their latest Chromium-based version installed.
          Information about it can be found here: <a href="https://www.microsoft.com/en-us/edge">Edge</a>.
        </p>
        <h3>Safari</h3>
        <p>
          The extension works in Safari 14, which is available on macOS 10.14 and above (Mojave, Catalina, Big Sur).
          To install, follow the <a href="https://apps.apple.com/us/app/gix-for-safari/id1543199063">App Store link</a> and
          allow your browser to open the link at the App Store, then Install the extension. When you open Safari it will
          display a notification with instructions on how to activate the extension. Once activated, you'll
          then need to click on the extension's icon from the toolbar and grant it access to read the web sites you
          visit (ideally "Always allow on every website"). Finally, you'll need to refresh the page you are on
          or navigate to a new page.
        </p>
        <h3>Chromium browsers</h3>
        <p>
          In addition to working with Chrome, the extension can work in any browser built
          on top of Chromium if the developers of that browser provide support for
          extensions. For a complete list of Chromium browsers, see
          <a href="https://en.wikipedia.org/wiki/Chromium_(web_browser)#Browsers_based_on_Chromium">Chromium
            browsers</a>.
          Other browsers in this category the extension is known to work in are Brave and Opera.
        </p>
        <p>
          To add the extension to Brave, simply go to the
          <a href="https://chrome.google.com/webstore/detail/gene-info/jggendahejbhkghnachhlomkkheomchp">
            Chrome Web Store</a> and click the "Add to Brave" button.
        </p>
        <p>
          To add the extension to Opera, first go to the
          <a href="https://chrome.google.com/webstore/detail/gene-info/jggendahejbhkghnachhlomkkheomchp">
            Chrome Web Store</a>. Click the "Add to Opera" button. A notification
            will appear at the top of the page asking if you want to install extensions from the
            Chrome Web Store. Confirm that you do.
        </p>
      </section>
      <section>
        <span class="anchor" id="manuscript"></span>
        <h2>Manuscript & citation</h2>
        <p>
          If you find the extension helpful and would like to cite us, please
          reference our manuscript published in
          <a href="https://www.nature.com/articles/s41592-019-0477-9">Nature Methods</a>.
        </p>
        <ul>
          <li><a href="https://www.ncbi.nlm.nih.gov/pubmed/31217594">PubMed</a></li>
          <li><a href="https://www.biorxiv.org/content/10.1101/532523v3">bioR&chi;iv</a></li>
        </ul>
      </section>
      <section>
        <span class="anchor" id="funding"></span>
        <h2>Funding</h2>
        <p>
          This project was supported by funding from
          <a href="http://www.cihr-irsc.gc.ca">CIHR</a>,
          <a href="https://www.computecanada.ca/">Compute Canada</a>,
          <a href="https://www.genomecanada.ca">Genome Canada</a>,
          <a href="https://www.nih.gov">National Institutes of Health</a>,
          <a href="http://www.ontariogenomics.ca">Ontario Genomics</a> and the
          <a href="https://www.ontario.ca/page/ontario-research-fund">Ontario Research Fund</a>.
        </p>
        <div class="funding-images">
          <img alt="CIHR" height="50" src="/assets/cihr.svg">
          <img
            alt="Compute Canada"
            height="50"
            src="/assets/compute-canada.png"
            srcset="/assets/compute-canada.png 1x,
              /assets/compute-canada2x.png 2x,
              /assets/compute-canada3x.png 3x"
          />
          <img alt="Genome Canada" height="42" src="/assets/genome-canada.svg">
          <img
            alt="National Institutes of Health"
            height="42"
            src="/assets/nih.png"
            srcset="/assets/nih.png 1x,
              /assets/nih2x.png 2x,
              /assets/nih3x.png 3x"
          />
          <img alt="Ontario genomics" height="42" src="/assets/ontario-genomics.svg">
          <img
            alt="Ontario Research Fund"
            height="42"
            src="/assets/orf.png"
            srcset="/assets/orf.png 1x,
              /assets/orf2x.png 2x,
              /assets/orf3x.png 3x"
          />
        </div>
      </section>
    </main>
    <footer>
      <small>© 2022 Gingras lab.</small>
    </footer>
  </div>
</body>

</html>
