<!doctype html>
<html lang="en">

<head>
  <title>Gene Info eXtension (GIX)</title>
  <meta name="Description" content="Gene Info eXtension (GIX) is a browser extension for retrieving gene information on any webpage.">
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme-color" content="#658170">
  <meta name="google-site-verification" content="_wWGZ3N0eO2YUHJv82oMCd1ikQXhMNMH-vPdXouEMcI" />

  <link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png">
  <link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png">
  <link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
  <link rel="manifest" href="site.webmanifest">
  <link rel="mask-icon" href="safari-pinned-tab.svg" color="#658170">
  <meta name="msapplication-TileColor" content="#fdfdfd">
  <meta name="theme-color" content="#658170">

  <link rel="stylesheet" href="assets/style.63edc9ac.css">
</head>

<body>
  <nav class="nav">
    <div class="nav__inner">
      <h1>GIX</h1>
      <ul class="nav__links">
        <li><a href="#video">Video tutorial</a></li>
        <li><a href="#settings">Settings</a></li>
        <li><a href="#options">Information options</a></li>
        <li><a href="#panel">Information panel</a></li>
        <li><a href="#database">Database</a></li>
        <li><a href="#unsupported">Unsupported websites</a></li>
        <li><a href="#permissions">Permissions</a></li>
        <li><a href="#logging">Logging</a></li>
        <li><a href="#source">Source code</a></li>
        <li><a href="#issues">Issues</a></li>
        <li><a href="#browser">Browser support</a></li>
        <li><a href="#funding">Funding</a></li>
        <li><a href="/workspace">Workspace</a></li>
      </ul>
      <hr>
      <small>
        Version 1.1.0
      </small>
    </div>
  </nav>
  <div class="documentation">
    <header>
      <h1>Gene Info eXtension (GIX)</h1>
      <p>
        GIX is a browser extensions that allows you to retrieve information about
        a gene product directly on any webpage simply by double clicking an official
        gene name, synonym or supported accession.
      </p>
      <p>
        This extension is developed by the laboratory of Dr. Anne-Claude Gingras at the
        Lunenfeld-Tanenbaum Research Institute in Toronto, Canada.
      </p>
      <p class="no-print">
        <a class="link-button" href="https://chrome.google.com/webstore/detail/gene-info/jggendahejbhkghnachhlomkkheomchp?hl=en-US">
          Download
        </a>
        from the Chrome Web Store.
      </p>
    </header>
    <main>
      <section class="documentation__links no-print">
        <h2>Contents</h2>
        <ul>
          <li><a href="#video">Video tutorial</a></li>
          <li><a href="#settings">Settings</a></li>
          <li><a href="#options">Information options</a></li>
          <li><a href="#panel">Information panel</a></li>
          <li><a href="#database">Database</a></li>
          <li><a href="#unsupported">Unsupported websites</a></li>
          <li><a href="#permissions">Permissions</a></li>
          <li><a href="#logging">Logging</a></li>
          <li><a href="#source">Source code</a></li>
          <li><a href="#issues">Issues</a></li>
          <li><a href="#browser">Browser support</a></li>
          <li><a href="#funding">Funding</a></li>
          <li><a href="/workspace">Workspace</a></li>
        </ul>
      </section>
      <section class="new-features">
        <h2>New features (Version 1.1.0)</h2>
        <ul>
          <li>
            Users can prevent the extension from activating except when the control
            (Linux/Windows) or command (Mac) keys are pressed through the "CTRL/⌘ required"
            option
          </li>
          <li>
            The order of information on the results panel can be set by dragging and
            dropping the settings on the extension popup
          </li>
          <li>
            There is now a dark theme in addition to the light theme that will apply
            to both the information panel and the extension popup
          </li>
          <li>
            Click the arrows on the interactor table to sort by gene or evidence numbers
          </li>
          <li>
            We've created an online workspace where users can paste lists of genes or text
            of any kind to make it easy to retrieve information on genes that may be contained
            in excel or word documents for example. A link to the workspace can be found
            at the bottom of the extension popup or at
            <a href="https://gene-info.org/workspace">gene-info.org/workspace</a>
          </li>
          <li>
            Pathology and pathway data from UniProt and Reactome respectively are now
            included on the information panel
          </li>
          <li>
            Users will be offered links to open search results at various sites when the
            extension is not able to find a match to a queried term 
          </li>
        </ul>
      </section>
      <section class="no-print" id="video">
        <h2>Video tutorial</h2>
        <video controls preload="metadata">
          <source src="media/tutorial.mp4" type="video/mp4">
          Sorry, your browser doesn't support embedded videos.
        </video>
      </section>
      <section id="settings">
        <h2>Settings</h2>
        <h3>Activation method</h3>
        <p>
          The default method for querying a gene is to double click on it. You can instead
          choose the drag method that requires you to mouse down on one side of the gene
          name, drag the selection box to the other side and release the mouse button. This
          can be useful on websites where double clicking selects more that one word (this
          is a rare occurrence and depends on the markup used by the website). The drag
          method is also necessary for gene names that have certain special characters in
          them. For example, double clicking a gene name with a dash '-' will only capture
          text before or after the dash but not including it.
        </p>
        <p>
          If you enable the "CTRL/⌘ required" option then double-clicking
          (or dragging) on its own will not initiate a query. On Linux and Windows systems
          you will have to hold the CTRL key while clicking to perform a query, or on Mac
          it is the Command/⌘ key. This restriction can ensure that the extension only activates
          when you want, rather than whenever you double-click.
        </p>
        <p>
          If you wish to completely disable the extension, simply click the disable button.
        </p>
        <h3>Display options</h3>
        <p>
          The extension can display information in two ways:
          <ol>
            <li>
              Detailed report: a panel spanning the full height of the browser window
              displaying links to external resources and the information from those resources.
            </li>
            <li>
              Tooltip report: a small box that opens at the point of the mouse click with
              only links to external resources.
            </li>
          </ol>
        </p>
        <h3>Species</h3>
        <p>
          In addition to human gene information our database contains information for
          the following model organisms:
        </p>
        <ul>
          <li><i>Arabidopsis thaliana</i></li>
          <li><i>Caenorhabditis elegans</i></li>
          <li><i>Danio rerio</i></li>
          <li><i>Dictyostelium discoideum</i></li>
          <li><i>Drosophila melanogaster</i></li>
          <li><i>Gallus gallus</i></li>
          <li><i>Mus musculus</i></li>
          <li><i>Saccharomyces cerevisiae</i></li>
          <li><i>Schizosaccharomyces pombe</i></li>
          <li><i>Xenopus laevis</i></li>
        </ul>
        <p>
          Some information will only be available for certain species. For instance, Human
          Protein Atlas localization and expression information is only available for human
          gene entries. The Compartments subcellular localization database has information
          on six of the organisms we support:
          <i>Arabidopsis</i>,
          <i>C. elegans</i>,
          <i>Drosophila</i>,
          human,
          mouse
          and yeast.
        </p>
        <h3>Query type</h3>
        <p>
          The extension can retrieve information for any of the following query types:
        </p>
        <ul>
          <li>Ensembl gene (e.g. ENSG00000114209)</li>
          <li>Ensembl protein (e.g. ENSP00000376506)</li>
          <li>Gene name (e.g. PDCD10)</li>
          <li>NCBI Entrez gene ID (e.g. 11235)</li>
          <li>neXtProt (e.g. NX_Q9BUL8)</li>
          <li>RefSeq (either nucleotide or protein e.g. NM_007217.3, NP_009148.2)</li>
          <li>UniProt (e.g. Q9BUL8)</li>
        </ul>
        <p>
          By default it will autodetect the query type but you can disable this option and
          force it to treat all queries as a specific type.
        </p>
        <p>
          Genes may also have a locus and ORF associated with them that can be used for querying.
          These were specifically included for yeast and flies respectively to enable querying by
          commonly used nomenclature of the following formats:
        </p>
        <ul>
          <li>YGL190C (yeast locus for gene CDC55)</li>
          <li>CG12217 (fly ORF for gene PpV)</li>
        </ul>
        <p>
          You can query by locus and ORF for other species, however it will only autodetect these
          query types in yeast and flies respectively as these species adhere to specific naming
          formats that do not hold in other species.
        </p>
        <h3>Search</h3>
        <p>
          The search box can be used for making queries in addition to using the mouse.
          Simply type in a query, hit enter and the information panel will appear on the
          current web page. Unlike mouse events, the search box will work on websites
          with embedded content like Google Docs and PDF viewers.
        </p>
      </section>
      <section id="options">
        <h2>Information options</h2>
        <p>
          Each pierce of information provided by GIX can be toggled off in the extension
          popup and will be hidden on subsequent queries.
        </p>
        <p>
          The GO term, Localization and RNA expression sections have advanced display
          options that can be viewed by clicking the settings/ellipsis button
          (<svg class="ellipsis" xmlns="http://www.w3.org/2000/svg" height="15" viewBox="0 0 512 512">
            <path fill="#658170" d="M328 256c0 39.8-32.2 72-72 72s-72-32.2-72-72 32.2-72 72-72 72 32.2 72 72zm104-72c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72zm-352 0c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72z" />
          </svg>)
          next to the relevant field in the extension popup. For GO terms, you can set the default
          namespace to display on the information panel: BP (biological process), CC
          (cellular compartment) or MF (molecular function). You can choose which
          resources to display for localization, including the Compartments subcellular
          localization database, the Human Protein Atlas and/or UniProt. Under RNA expression
          you can choose which cell lines and tissues to display by default in the information
          panel. There are 64 cell lines and 37 tissue types to choose from that we have
          incorporated from the Human Protein Atlas.
        </p>
        <p>
          You can control the order in which information appears on the results panel
          by dragging and dropping the settings in the extension popup.
        </p>
        <p>
          Certain data is only available for specific species as outlined above
          in the <strong>Species</strong> section.
        </p>
      </section>
      <section id="panel">
        <h2>Information panel</h2>
        <h3>Multiple results</h3>
        <p>
          If a query matches to multiple genes, as can happen since gene names are not always unique,
          the information panel will
          display one result and the others will be available from the dropdown menu
          that appears next to the <strong>Gene</strong> heading. A match to an
          official gene name will always be selected at the initial result to display
          in the case of multiple matches.
        </p>
        <h3>Buttons</h3>
        <ul class="buttons">
          <li>
            <img alt="Close button" src="assets/close-button.svg">
            close the extension
          </li>
          <li>
            <img alt="Drag button" src="assets/drag-button.svg">
            reposition the extension horizontally
          </li>
          <li>
            <img alt="Back button" src="assets/back-button.svg">
            go back to the previous result
          </li>
          <li>
            <img alt="Resize button" src="assets/resize-button.svg">
            resize the extension (bottom left corner)
          </li>
        </ul>
      </section>
      <section id="database">
        <h2>Database</h2>
        <p>
          The database will be updated on the 15th of each month or the next
          business day if the 15th falls on a weekend or holiday.
        </p>
        <h3>Integrated resources</h3>
        <p>
          We integrate several databases to generate the database used for Gene Info.
          The resources, urls and citations are listed below in alphabetical order:
        </p>
        <ul>
          <li>
            BioGRID,
            <a href="https://thebiogrid.org">https://thebiogrid.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/16381927">PMID:16381927</a>)
          </li>
          <li>
            Compartments,
            <a href="https://compartments.jensenlab.org">https://compartments.jensenlab.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/24573882">PMID:24573882</a>)
          </li>
          <li>
            Gene Ontology,
            <a href="http://www.geneontology.org">http://www.geneontology.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/10802651">PMID:10802651</a>,
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/27899567">PMID:27899567</a>)
          </li>
          <li>
            HGNC,
            <a href="https://www.genenames.org">https://www.genenames.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/27799471">PMID:27799471</a>)
          </li>
          <li>
            Human Protein Atlas,
            <a href="https://www.proteinatlas.org">https://www.proteinatlas.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/25613900">PMID:25613900</a>,
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/28495876">PMID:28495876</a>)
          </li>
          <li>
            IntAct,
            <a href="https://www.ebi.ac.uk/intact">https://www.ebi.ac.uk/intact</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/24234451">PMID:24234451</a>)
          </li>
          <li>
            OMIM,
            <a href="https://www.omim.org">https://www.omim.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/30445645">PMID:30445645</a>)
          </li>
          <li>
            Pfam,
            <a href="https://pfam.xfam.org">https://pfam.xfam.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/30357350">PMID:30357350</a>)
          </li>
          <li>
            Reactome,
            <a href="https://reactome.org">https://reactome.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/29145629">PMID:29145629</a>)
          </li>
          <li>
            UniProt,
            <a href=" https://www.uniprot.org"> https://www.uniprot.org</a>
            (<a href="https://www.ncbi.nlm.nih.gov/pubmed/29425356">PMID:29425356</a>)
          </li>
        </ul>
      </section>
      <section id="unsupported">
        <h2>Unsupported websites</h2>
        <p>
          The extension is not able to capture mouse events (double clicks, etc.)
          on websites with embedded elements like Google Docs and PDF viewers and you
          can not therefore perform queries in the default way. The search box in the extension
          popup can however still be used for performing queries on these sites.
        </p>
        <p>
          Chrome blocks extensions from running on the tab homepage - the page you see when
          opening a new tab - as well as a few other privileged pages, and so neither double
          clicking nor the search bar will work in these contexts.
        </p>
      </section>
      <section id="permissions">
        <h2>Permissions</h2>
        <p>
          When you install the extension it will alert you that it can "Read and change
          all your data on the websites you visit".
        </p>
        <p>
          To capture text from websites you visit this extension needs to be granted
          permission to read the sites. The only thing this extension reads is the
          text selected via double clicking or mouse dragging (if you enable that
          option).
        </p>
        <p>
          Adding an information panel or tooltip to a webpage with information about the gene you have
          selected requires permission to modify or "change all your data on the websites
          you visit". The only thing we modify about the websites you visit is the addition of the
          panel or tooltip.
        </p>
        <p>
          If you would like to further restrict the access given to this extension
          you can right click on the icon in your browser toolbar and scroll to
          "This Can Read and Change Site Data" to view the options. Detailed
          instructions can be found at the
          <a href="https://support.google.com/chrome_webstore/answer/2664769?p=enable_extensions&visit_id=636834215946361040-2354360077&rd=1#extensionpermissions">
            Web Store's Help Page</a>.
        </p>
      </section>
      <section id="logging">
        <h2>Logging</h2>
        <p>
          We log three pieces of information with each query: 1) whether it matched to
          something in our database; 2) the species; and 3) the type of query, i.e. was
          it a gene name, UniProt ID, etc. We do not log the actual query term itself.
          Recording the total number of queries allows us to ensure the server has
          enough hardware resources to meet the demands of the user base. Recording the
          species allows us to prioritize development time and add resources for the
          most used species. For example, if a model organism is only searched by 0.01%
          of the user base, it will get less priority than one used by 95% of users. Finally,
          recording the type of query allows us to determine whether it is worth continuing to
          support a particular type of accession.
        </p>
        <p>
          We do not record an personal information or IP addresses and the extension does
          not use analytics.
        </p>
      </section>
      <section id="source">
        <h2>Source code</h2>
        <p>
          Source code is available at
          <a href="https://github.com/knightjdr/gene-info/">GitHub</a>.
        </p>
      </section>
      <section id="issues">
        <h2>Issues</h2>
        <p>
          If you find the extension does not work for you it may be that you are located
          behind a very strict firewall that blocks the method we are using to fetch data
          from our server (XMLHttpRequest) or that blocks the requests but not the method
          specifically. If this is the case, you'll have to discuss
          the matter with your IT department. If we can help to resolve the issue, please
          contact us.
        </p>
        <p>
          If you would like to report a bug, request a feature or have your resource
          added to Gene Info, please use our
          <a href="https://github.com/knightjdr/gene-info/issues">issue tracker</a>.
        </p>
      </section>
      <section id="browser">
        <h2>Browser support</h2>
        <p>
          In addition to working with Chrome the extension can work in any browser built
          on top of Chromium if the developers of that browser provide support for
          extensions. For a complete list of Chromium browsers, see
          <a href="https://en.wikipedia.org/wiki/Chromium_(web_browser)#Other_browsers_based_on_Chromium">Chromium
            browsers</a>.
          Currently the extension is known to work in the Brave browser.
        </p>
        <p>
          To add an extension to Brave simply go to the Chrome Web Store, find the extension
          and click the "Add to Chrome" button.
        </p>
        <p>
          We are currently working on porting the extension to Firefox and Safari but
          cannot provide an ETA.
        </p>
      </section>
      <section id="funding">
        <h2>Funding</h2>
        <p>
          This project was supported by funding from
          <a href="http://www.cihr-irsc.gc.ca">CIHR</a>,
          <a href="https://www.genomecanada.ca">Genome Canada</a>,
          <a href="https://www.nih.gov">National Institutes of Health</a>,
          <a href="http://www.ontariogenomics.ca">Ontario Genomics</a> and the
          <a href="https://www.ontario.ca/page/ontario-research-fund">Ontario Research Fund</a>.
        </p>
        <div class="funding-images">
          <img alt="CIHR" height="50" src="assets/cihr.svg">
          <img alt="Genome Canada" height="42" src="assets/genome-canada.svg">
          <img alt="National Institutes of Health" height="42" src="assets/nih.png" srcset="
              assets/nih.png 1x,
              assets/nih2x.png 2x,
              assets/nih3x.png 3x"
            width={146} />
          <img alt="Ontario genomics" height="42" src="assets/ontario-genomics.svg">
          <img alt="Ontario Research Fund" height="42" src="assets/orf.png" srcset="
              assets/orf.png 1x,
              assets/orf2x.png 2x,
              assets/orf3x.png 3x"
            width={146} />
        </div>
      </section>
    </main>
    <footer>
      <small>© 2019 Gingras lab.</small>
    </footer>
  </div>
</body>

</html>